pulp_smash/tests/puppet/api_v2/test_sync_publish.py

```diff
@@ -76,6 +82,40 @@ def _gen_repo():
         'notes': {'_repo-type': 'puppet-repo'},
     }
 
+class _CleanUpTestCaseMixin(object):
+    """Provide functions to clean up database and filesystem of Pulp"""
+
+    @classmethod
+    def reset_database(cls, server_config):
+        """Drop database and re-run migrations."""
+        cls.services = tuple((
+            cli.Service(cls.cfg, service) for service in _SERVICES
+        ))
+
+        # Stop all Pulp services
+        for service in cls.services:
+            service.stop()
+
+        # Drop database and re-run migrations
+        client = cli.Client(server_config, cli.echo_handler)
+        drop_db_command = ('mongo', 'pulp_database', '--eval', 'db.dropDatabase()')
+        client.run(drop_db_command)
+        migrations_command = ('sudo', '-u', 'apache', 'pulp-manage-db')
+        client.run(migrations_command)
+
+        # Restart all Pulp services
+        for service in cls.services:
+            service.start()
+
+    @classmethod
+    def clean_filesystem(cls, server_config):
+        """Remove all content from /var/lib/pulp/[content, published]"""
+        client = cli.Client(server_config, cli.echo_handler)
+        content_command = ('rm', '-rf', '/var/lib/pulp/content/*')
+        published_command = ('rm', '-rf', '/var/lib/pulp/published/*')
+        client.run(content_command)
+        client.run(published_command)
```

Comment:
This can also be written as:

```python
client.run('rm -rf /var/lib/pulp/content/*'.split())
client.run('rm -rf /var/lib/pulp/published/*'.split())
```